controllers/inverse_kinematics_pyikfast/inverse_kinematics_pyikfast.py

```diff
@@ -44,20 +51,13 @@
 # -------------------- INITIALIZATION -----------------------
 
 # Initialize the Webots Supervisor.
-supervisor = Supervisor()
+print("getTypeName", supervisor.getSelf().getTypeName())
 if not supervisor.getSupervisor():
-    sys.exit('WARNING: Your robot is not a supervisor! Set the supervisor field to True and restart the controller.')
+    sys.exit(
+        "WARNING: Your robot is not a supervisor! Set the supervisor field to True and restart the controller."
+    )
 timeStep = int(supervisor.getBasicTimeStep())
 
-
-# check if our world already has the TARGET node. If not, we spawn it.
-target = supervisor.getFromDef('TARGET')
-try:
-    target.getPosition()
-except Exception as e:
-    print('No TARGET defined. Spawning TARGET sphere')
-    spawnTarget(supervisor)
-
```

Comment:
Why delete the TARGET sphere check? If someone implements the controller in their own world, the target sphere should be spawned.